src/Dashboard/Grid.php

```diff
@@ -1442,11 +1443,12 @@ public function getAllDasboardCards($force = false): array
                     'content' => __("Toggle edit mode to edit content"),
                 ]
             ];
+            $GLPI_CACHE->set('getAllDasboardCards', $cards, new DateInterval('P1D'));
```

Comment:
Could probably be cached without time restriction. Unless GLPI is updated, there is no reason to have a different result, and when GLPI is updated, cache is cleared anyway.

```suggestion
            $GLPI_CACHE->set('getAllDasboardCards', $cards);
```

Reply:
Done.